freqtrade/strategy/interface.py

```diff
@@ -19,30 +20,74 @@ class IStrategy(ABC):
         ticker_interval -> str: value of the ticker interval to use for the strategy
     """
 
+    # associated minimal roi
     minimal_roi: Dict
+
+    # associated stoploss
     stoploss: float
+
+    # associated ticker interval
     ticker_interval: str
 
-    @abstractmethod
     def populate_indicators(self, dataframe: DataFrame) -> DataFrame:
         """
         Populate indicators that will be used in the Buy and Sell strategy
         :param dataframe: Raw data from the exchange and parsed by parse_ticker_dataframe()
         :return: a Dataframe with all mandatory indicators for the strategies
         """
+        warnings.warn("deprecated - please replace this method with advise_indicators!",
+                      DeprecationWarning)
+        return dataframe
 
-    @abstractmethod
     def populate_buy_trend(self, dataframe: DataFrame) -> DataFrame:
         """
         Based on TA indicators, populates the buy signal for the given dataframe
         :param dataframe: DataFrame
         :return: DataFrame with buy column
         """
+        warnings.warn("deprecated - please replace this method with advise_buy!",
+                      DeprecationWarning)
+        dataframe.loc[(), 'buy'] = 0
+        return dataframe
 
-    @abstractmethod
```

Comment:
Assuming this is still thought to be a abstract class ... why remove the `@abstractmethod` decorator?

Reply:
to be clear about this - if there's a good reason i'm fine with it - i just don't understand it at the moment ... :smiley:

Reply:
mhm good point, ok lets leave it in.